recipes/orange-widget-base/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "orange-widget-base" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "0f080a410f6aa9c929d8f7f22b7157ac3d53ca0d34de10026dedaaedad46b8e9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib
+    - pyqtgraph
+    - anyqt
+    - orange-canvas-core
+
+test:
+  imports:
+    - orangewidget.widget
+    - orangewidget.gui
+
+about:
+  home: https://github.com/biolab/orange-widget-base
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
+  summary: 'Base Widget for Orange Canvas'
+
+  description: |
+    This project implements the base OWBaseWidget class and utilities for use
+    in Orange Canvas workflows.
+
+  doc_url: https://orange-widget-base.readthedocs.io/
+  dev_url: https://github.com/biolab/orange-widget-base/
+
+extra:
+  recipe-maintainers:
+    - ales-erjavec
+    - lanzagar
+    - markotoplak
+    - primozgodec
```

Comment:
@lanzagar @markotoplak @primozgodec can you confirm you would like to be maintainers of this recipe

Reply:
I confirm.

Reply:
I confirm

Reply:
I confirm.